molgenis-core-ui/src/main/java/org/molgenis/ui/I18nStringsPopulator.java

```diff
@@ -33,26 +34,50 @@
 
 	private final FileRepositoryCollectionFactory fileRepositoryCollectionFactory;
 	private final ImportServiceFactory importServiceFactory;
+	private final FileStore fileStore;
 
 	@Autowired
 	public I18nStringsPopulator(FileRepositoryCollectionFactory fileRepositoryCollectionFactory,
-			ImportServiceFactory importServiceFactory)
+			ImportServiceFactory importServiceFactory, FileStore fileStore)
 	{
 		this.fileRepositoryCollectionFactory = fileRepositoryCollectionFactory;
 		this.importServiceFactory = importServiceFactory;
+		this.fileStore = fileStore;
 	}
 
 	@Override
 	public void onApplicationEvent(ContextRefreshedEvent event)
 	{
 		LOG.info("Importing i18n strings...");
+		final String i18nFileName = "i18n.xlsx";
+
+		// "i18n is saved as a Application/Library resource.
+		// "It is not possible to use it as a file but should streamed as a resource"
+		ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+		InputStream is = classLoader.getResourceAsStream(i18nFileName);
```

Comment:
Look at (Warning for users of Tomcat 7 and below) in: http://stackoverflow.com/questions/676250/different-ways-of-loading-a-file-as-an-inputstream 

It should be the same but it is not.